scripts/wikitables/evaluator.py

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/env python
```

Comment:
This needs to go inside `allennlp`, so that it can be used in models (and so that it gets distributed via pip installs, etc.).  We currently have `squad_eval.py` in `allennlp.common`, and `srl-eval.pl` in `allennlp.tools`.  I'd say this should go in `allennlp.tools` (and `squad_eval.py` should probably be moved there).

Reply:
Done.